modules/luci-base/htdocs/luci-static/resources/network.js

```diff
@@ -3861,12 +3861,17 @@ WifiNetwork = baseclass.extend(/** @lends LuCI.network.WifiNetwork.prototype */
 		var mode = this.getActiveMode();
 
 		switch (mode) {
-		case 'Master':  return _('Master');
-		case 'Client':  return _('Client');
-		case 'Ad-Hoc':  return _('Ad-Hoc');
-		case 'Mash':    return _('Mesh');
-		case 'Monitor': return _('Monitor');
-		default:        return mode;
+		case "Master":       return _('Access Point');
```

Comment:
nitpick, but do the `'` really need to change to `"` in the JS?

Reply:
yep, slip of the tongue. fixed